lib/util/exec/containerbase.ts

```diff
@@ -65,6 +65,11 @@ const allToolConfig: Record<string, ToolConfig> = {
     depName: 'helm/helm',
     versioning: semverVersioningId,
   },
+  helmfile: {
```

Comment:
this needs a new installer inside `containerbase/buildpack`[^1] image.

[^1]: https://github.com/containerbase/base

Reply:
Made a PR for this in https://github.com/containerbase/base/pull/674